command_node:
  ros__parameters:
    topic_name: "command"
    controller_name: "omni_controller/"
    set_point_list:
      - start
      - move_for
      - straight
      - slow_down
      - turn
      - move_for1
      - straight1
      - slow_down1
      - turn1
      - move_for2
      - straight2
      - slow_down2
      - stop
    start:
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 0.0
    move_for:
      vx: -0.15
      vy: 0.0
      omega: 0.0
      time: 1.0
    straight:
      vx: -0.15
      vy: 0.0
      omega: 0.0
      time: 20.0
    slow_down:
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 21.0
    turn:
      vx: 0.0
      vy: 0.0
      omega: 1.5
      time: 24.142
    move_for1:
      vx: -0.15
      vy: 0.0
      omega: 0.0
      time: 25.142
    straight1:
      vx: -0.15
      vy: 0.0
      omega: 0.0
      time: 45.142
    slow_down1:
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 46.142
    turn1:
      vx: 0.0
      vy: 0.0
      omega: 1.5
      time: 49.284
    move_for2:
      vx: -0.15
      vy: 0.0
      omega: 0.0
      time: 50.284
    straight2:
      vx: -0.15
      vy: 0.0
      omega: 0.0
      time: 70.284
    slow_down2:
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 71.284
    stop:
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 75.0
